README.md

```diff
@@ -33,65 +32,198 @@ Only time will tell if making our product roadmap
+we will need to _significantly_ leverage
+the Open Source work of many brilliant people.
+But we are getting ahead of ourselves;
+we need to _first_ focus
+on creating "MVP" versions
+of each step in the workflow.
+
+For the _second_ step, ***Categorise***,
+we will need a tagging system that allows people
+to apply textual labels to items.
+The UX for this is still "TBD".
+We will need to sketch it out ahead of sprint planning.
+See: https://github.com/dwyl/product-roadmap/issues/15
+
+
+And finally for the _third_ step, ***Complete***,
+we will build a simple checklist system.
+See: [`checklists.md`](https://github.com/dwyl/product-roadmap/blob/master/checklists.md)
+
+
+
+
+
+### USP: What is _different_ about the @dwyl App?
+
+A valid/relevant **question** is: what makes the @dwyl App unique?
+i.e. What is our
+["unique selling point"](https://en.wikipedia.org/wiki/Unique_selling_proposition)?<br />
+The **answer** is: _initially_ it may _appear_ that our App
+does not have much different
+from existing apps that _partially_ solve the personal effectiveness challenge.
+The _major_ differences will become apparent in due course:
+1. **People Not _Profits_** - We are playing an
+["***infinite game***"](https://github.com/dwyl/start-here/issues/190)
+where our _mission_ is to
+"_Empower people to maximise effectiveness, creativity and happiness_."
+See:
+[What is dwyl's mission?](https://github.com/dwyl/start-here/blob/master/mission.md#what-is-dwyls-mission)
+We will not _rest_ until we have created a system/app
+that is _universally accessible_ to every human being
+that allows _everyone_ a fair opportunity to live their best life.
+We will _cooperate_ with _many_ people/organisations
+to build a comprehensive solution that _anyone_ can use.
+2. ***Open Source*** - All of our code will _always_ be ***Open Source***.
+That means anyone can modify/extend/improve it.
+And if people feel we are not doing a good enough job
+of stewarding the App/community, they can _tell_ us!
+Of the _many_ "productivity" apps out there,
+almost _none_ are Open Source.
+This means that your usage is locked-in to their system for better or worse.
+Your data is bound by their "terms and conditions"
+(_including being visible to 3<sup>rd</sup> party "analytics" providers
+i.e. Google_)
+and you are a _hostage_ to their whims.
+If they decide to have a UI re-write
+that
+[_completely_ changes](https://github.com/dwyl/product-roadmap/issues/14)
+how you interact with the App,
+you have _no control_ and are _forced_ to use the "new version";
+their way or the highway.
+We are going to _eliminate forced updates_
+by implementing a UX/UI innovation we call Progressive UI.
+3. **Progressive UX/UI** - the UX/UI for _each_ person using the app,
+will be _personalised_ to the individual
+based on the features they (_most_) _use_.
+If the person does not _use_ a particular function, they won't _see_ it.
+> Note: This is might not feel relevant during "MVP",
```

Comment:
🙌 This will of course be coupled with a discovery section where people can take a look at other features

Reply:
Agreed. That's going to be one of our USPs. Each _feature_ will have a walkthrough demo/video that will explain _why_ the feature exists, _who_ it's for and _how_ to use it. 💡